### ISSUE TITLE
Combobox behaves strangely after 13.0.5

### ISSUE THREAD

**Issue by ericmartinezr** [vaadin/vaadin-combo-box-flow#250]
After upgrading from 13.0.0 to 13.0.5 combobox started to behave strangely. When you open a combobox, intially you see one or two items, and 1 second later the rest of the items show up. From there everything works correctly.

The code it's simple and to be honest I don't know how to provide a better reproduction (something runnable)

```java
ComboBox<Bean> cbox = new ComboBox<>();
cbox.setItems(beanRepository.findAll());
```

It's probably related to this issue https://github.com/vaadin/vaadin-combo-box-flow/issues/222, but I can't confirm it

I have [this recording](https://drive.google.com/file/d/1TwkRPOQdoLAqL4rPedclxEs8be2pZ6SI/view) I provided in gitter, where I had a confirmation from another user https://gitter.im/vaadin/flow?at=5cc849fd6a84d76ed81ef553

Hopefully I made the issue clear. If you need more info let me know.

**Comment by KasparScherrer** [comment]
I confirm this behaviour, it seems to be freshly introduced with the version 13.0.5 as I do not experience this with 13.0.4

**Comment by web-padawan** [comment]
This is a result of the performance fix targeting Edge: vaadin/vaadin-combo-box#800

**Comment by pekam** [comment]
The fix https://github.com/vaadin/vaadin-combo-box/pull/814 will be included in the next web component release.
